lib/instrumentation/modules/@apollo/server.js

```diff
@@ -6,13 +6,22 @@
 
 'use strict';
 
+const semver = require('semver');
 const shimmer = require('../../shimmer');
 
-module.exports = function (apolloServer, agent, { enabled }) {
+module.exports = function (apolloServer, agent, { version, enabled }) {
   if (!enabled) {
     return apolloServer;
   }
 
+  if (!semver.satisfies(version, '^4.0.0')) {
+    agent.logger.debug(
+      '@apollo/server version %s not supported - aborting...',
```

Comment:
```suggestion
      '@apollo/server version %s not supported, skipping @apollo/server instrumentation',
```

"aborting..." in a log file looks worse that it is. It can suggest that the who APM agent itself is aborting.